lib/modules/datasource/metadata.ts

```diff
@@ -87,6 +87,14 @@ export function normalizeDate(input: any): string | null {
     //   2. Format of `input` is very exotic
     //      (from `DateTime.fromISO()` perspective)
     //
+    if (input.match(regEx(/^\d{14}$/))) {
```

Comment:
I don't think we need this, the fromISO below catches this. otherwise the go datasource would already filter them.

see other comments